app/services/plans/update_service.rb

```diff
@@ -89,6 +91,12 @@ def bill_charges_monthly?
       params[:bill_charges_monthly] || false
     end
 
+    def bill_fixed_charges_monthly?
+      return unless params[:interval]&.to_sym == :yearly
+
+      params[:bill_fixed_charges_monthly] || false
```

Comment:
```suggestion
      !!params[:bill_fixed_charges_monthly]
```

its more idiomatic perhaps.